### PR TITLE
SPCR-110 Add descriptive error summary to People and Vessel tabs

### PR DESCRIPTION
**Pull request by Adele-B**
## Description
Add descriptive error messages to the People and Vessel tab forms. Anchor the error messages to the div surrounding each input.

### To test
- Click on the people tab
- Click on 'save a person'
- Scroll to the bottom and click 'Add to saved people list'
- You should be taken to the error summary at top of page
- Click on the error messages
- They should take you to the top of the relevant inputs

- Repeats steps for the Vessel tab

## Developer Checklist

\* Required

- [x] Up-to-date with master branch? \*

- [ ] Does it have tests?

- [x] Pull request URL linked to JIRA ticket? \*

- [ ] All reviewer comments resolved/answered? \*

Ensure you delete the branch once the Pull Request is merged.

### CODE REVIEW

File: src/components/People/FormPerson.jsx [hunk 1]
```diff
@@ -39,14 +39,14 @@ const FormPerson = ({
         </label>
       </div>
 
-      <div className={`govuk-form-group ${errors.gender ? 'govuk-form-group--error' : ''}`}>
+      <div id='gender' className={`govuk-form-group ${errors.gender ? 'govuk-form-group--error' : ''}`}>
```
Comment (p3dr0migue1) [inline src/components/People/FormPerson.jsx:42]: Double quotes please

File: src/components/People/FormPerson.jsx [hunk 2]
```diff
@@ -181,15 +181,15 @@ const FormPerson = ({
         </label>
       </div>
 
-      <div className={`govuk-form-group ${errors.nationality ? 'govuk-form-group--error' : ''}`}>
+      <div id="nationality" className={`govuk-form-group ${errors.nationality ? 'govuk-form-group--error' : ''}`}>
```
Comment (p3dr0migue1) [inline src/components/People/FormPerson.jsx:184]: Double quotes on `"govuk-form-group--error"` and everywhere else below

**Review by p3dr0migue1 — CHANGES_REQUESTED** [review]

File: src/components/People/FormPerson.jsx (hunk 2 above)
Comment (Adele-B) [reply src/components/People/FormPerson.jsx:184]: My linter throws an error when I try to put "govuk-form-group--error" in double quotes - is this wrong?
Screenshot 2022-01-10 at 17 19 20

Comment (jennikate) [reply src/components/People/FormPerson.jsx:184]: @Adele-B it should be single quotes for string when it's within JavaScript, and double quotes when it's in html

Comment (jennikate) [reply src/components/People/FormPerson.jsx:184]: Let me know if your linter isn't supporting this as it's the default rule in the linters we use

**Review by jennikate — APPROVED** [review]
LGTM

**Pull request merged**